src/Neo/SmartContract/Native/LedgerContract.cs

```diff
@@ -47,6 +48,10 @@ internal override ContractTask OnPersist(ApplicationEngine engine)
             foreach (TransactionState tx in transactions)
             {
                 engine.Snapshot.Add(CreateStorageKey(Prefix_Transaction).Add(tx.Transaction.Hash), new StorageItem(tx));
+                foreach (var attr in tx.Transaction.GetAttributes<Conflicts>())
+                {
+                    engine.Snapshot.GetOrAdd(CreateStorageKey(Prefix_Transaction).Add(attr.Hash), () => new StorageItem(new TransactionState { Trimmed = true }));
```

Comment:
We can use a different key, but at the same time reusage of `TransactionState` allows us to search for a single `Prefix_Transaction` instead of searching for two prefixes in those places where it's required to retrieve either transaction or conflicting transaction mock. Are you sure we still want to use a different key for this?

Reply:
Solved in 8f53e90f708069af4a6ca887b36d1efcd4eff0c1.